tests/config_dpb_test.py

```diff
@@ -300,27 +417,46 @@ def test_config_breakout_negative_cases(self, sonic_db):
         runner = CliRunner()
         obj = {'config_db':db.cfgdb}
 
+        # Input Data
+        interface = 'Ethern'
+        curMode = '4x25G[10G]'
+        newMode = '2x50G'
+
+
         # Wrong interface name
         result = runner.invoke(config.config.commands["interface"].\
-            commands["breakout"], ['Ethern', '2x50G', '-v', '-y'], obj=obj)
+            commands["breakout"], ['{}'.format(interface), '{}'.format(newMode), '-v', '-y'], obj=obj)
+
         print(result.exit_code, result.output)
         assert result.exit_code == 1
         #TODO: Uncomment it after Dev Fix, right now it is python bt
```

Comment:
Looks like this is done with this commit. Remove this TODO comment?

Reply:
Removed